data/behavior_data.c

```diff
@@ -204,6 +204,12 @@ enum BehaviorCommands {
 #define OR_INT(field, value) \
     BC_BBH(BHV_CMD_OR_INT, field, value)
 
+// Performs a bitwise OR with the specified field and the given (32 bit) integer.
+// Usually used to set an object's flags which use values above 16 bits.
+#define OR_LONG(field, value) \
```

Comment:
I would actually move this down to a location that matches the enum location (i.e. above `SET_MODEL`).

Reply:
Alternatively, shift the enums around such that OR_LONG's location feels less random